source/index.html.md

```diff
@@ -540,26 +554,123 @@ Exactly one payment method must be used.
   <dd>
     Amount in minor units of given currency (e.g. cents if in Euro).
   </dd>
+
   <dt>currency
     <span class="type">[A-Z]{3}</span>
   </dt>
   <dd>
     <a target="_blank" href="currencies.txt">3-letter currency code</a>. (Some exponents differ from ISO 4217.)
   </dd>
+
+  <dt>credential
```

Comment:
```suggestion
  <dt>cof
```

Reply:
In no specific order:
1. `cof = (first|subsequent)`
2. `cof = (store|use)`
3. `cof = (storing|using)`

Reply:
I like the simplicity/brevity of 2, and I find the phrases "store credential of file" and "use credential on file" meaningful and clear :slightly_smiling_face:

Reply:
I have a 🥩 with `cof` — that it's a TLA that we need to explain. That's why I tried to come up with something that could compete, and my best attempt was if we are `storing [the] credential` or `[grabbing the] credential from_storage` (with `using` I was trying to communicate `using [stored] credential`, but the `stored` piece is pretty important word to describe the right intent).

Reply:
Migrated to https://github.com/clearhaus/gateway-api-docs/pull/111/files#r524296900.